torch/ao/quantization/qconfig_mapping.py

```diff
@@ -69,7 +69,7 @@ def _get_default_qconfig_mapping(is_qat: bool, backend: str, version: int) -> QC
     # so we have to modify the weight observer to default_weight_observer or another
     # per tensor supported observer.
     # see https://github.com/pytorch/pytorch/issues/47535
-    if backend == "fbgemm":
+    if backend in ("fbgemm", "x86"):
```

Comment:
just to clarify, we don't want `onednn` here?

Reply:
No. Onednn supports per-channel.